src/site/content/en/blog/sign-in-form-best-practices/index.md

```diff
@@ -50,9 +50,9 @@ Here is an example of a simple sign-in form that demonstrates all of the best pr
 * Use element attributes to [access built-in browser
```

Comment:
Another unrelated file was modified. This one is mostly OK because it's just removing trailing whitespace, but there is one thing we need to fix.

Reply:
I'm not sure what the change is here?